daemon/logger/factory.go

```diff
@@ -46,7 +53,7 @@ func (lf *logdriverFactory) get(name string) (Creator, error) {
 
 	c, ok := lf.registry[name]
 	if !ok {
-		return c, fmt.Errorf("logger: no log driver named '%s' is registered", name)
+		return c, fmt.Errorf("unsupported log driver '%s'", name)
```

Comment:
I think the error message `logger: no log driver named '%s' is registered` is a bit misleading, the user might ask "how do I register log drivers?"
But this change caused the `docker-py` test failing at https://github.com/docker/docker-py/blob/master/tests/integration/container_test.py#L288